server/kafka/mock_schema_registry.go

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2019-2022 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package kafka
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/riferrei/srclient"
+	"github.com/stretchr/testify/assert"
+)
+
+// schemas for testing
+const avroSchema = "{" +
+	"\"type\": \"record\"," +
+	"\"name\": \"snack\"," +
+	"\"fields\": [" +
+	"{\"name\": \"name\", \"type\": \"string\"}," +
+	"{\"name\": \"manufacturer\", \"type\": \"string\"}," +
+	"{\"name\": \"calories\", \"type\": \"float\"}," +
+	"{\"name\": \"color\", \"type\": [\"null\", \"string\"], \"default\": null}" +
+	"]}"
```

Comment:
We can use raw string literals (with backticks) here to make this a little more readable and writable.
```go
const avroSchema = `{
	"type": "record",
	"name": "snack",
	"fields": [
	{"name": "name", "type": "string"},
	{"name": "manufacturer", "type": "string"},
	{"name": "calories", "type": "float"},
	{"name": "color", "type": ["null", "string"], "default": null}
	]}`
```

Reply:
Done.